prototypes/uniformtree.py

```diff
@@ -434,6 +470,82 @@ def deleteRecursive(self, nodeKey):
 				continue
 			s.delete()
 
+	@exposed
+	@forceSSL
+	@forcePost
+	def move(self, skelType: str, key: str, parentNode: str, *args, **kwargs) -> str:
+		"""
+		Move a node (including its contents) or a leaf to another node.
+
+		.. seealso:: :func:`canMove`
+
+		:param skelType: Defines the type of the entry that should be moved and may either be "node" or "leaf".
+		:param key: URL-safe key of the item to be moved.
+		:param parentNode: URL-safe key of the destination node, which must be a node.
+
+		:returns: The rendered, edited object of the entry.
+
+		:raises: :exc:`viur.core.errors.NotFound`, when no entry with the given *key* was found.
+		:raises: :exc:`viur.core.errors.Unauthorized`, if the current user does not have the required permissions.
+		:raises: :exc:`viur.core.errors.PreconditionFailed`, if the *skey* could not be verified.
+		"""
+		if skelType == "node":
+			skelType = TreeType.Node
+		elif skelType == "leaf" and self.leafSkelCls:
+			skelType = TreeType.Leaf
+		else:
+			raise errors.NotAcceptable()
+
+		skel = self.addSkel(skelType)  # srcSkel - the skeleton to be moved
+		parentNodeSkel = self.editSkel(TreeType.Node)  # destSkel - the node it should be moved into
+
+		if not skel.fromDB(key) or not parentNodeSkel.fromDB(parentNode):
+			# Could not find one of the entities
+			raise errors.NotFound()
+
+		if not self.canMove(skelType, skel, parentNodeSkel):
```

Comment:
But _canMove_ expects here the skelType as str "node" oder "leaf"

Reply:
I think skelType should also be that enum

Reply:
Okay, but than we should change the docstring + type annotations of `canMove` to avoid misunderstandings.